tests/v2/test_1302-layout-builder-nested-list.py

```diff
@@ -4,117 +4,6 @@
 
 
-def test_nested_layout_builder():
-    form = """
-{
-    "class": "RecordArray",
-    "contents": {
-        "u": {
-            "class": "ListOffsetArray",
-            "offsets": "i64",
-            "content": {
-                "class": "RecordArray",
-                "contents": {
-                    "i": "int64",
-                    "j": {
-                        "class": "ListOffsetArray",
-                        "offsets": "i64",
-                        "content": "int64"
-                    }
-                }
-            }
-        },
-        "v": "int64",
-        "w": "int64",
-        "x": "int64"
-    }
-}
-"""
-    builder = ak._v2.highlevel.LayoutBuilder(form)
-    #
-    #     """
-    # {
-    #     "form_key": "root",
-    #     "class": "RecordArray",
-    #     "contents": {
-    #         "u": {
-    #             "form_key": "u",
-    #             "class": "ListOffsetArray",
-    #             "offsets": "i64",
-    #             "content": {
-    #                 "form_key": "u-content",
-    #                 "class": "RecordArray",
-    #                 "contents": {
-    #                     "i": {
-    #                         "class": "NumpyArray",
-    #                         "primitive": "int64",
-    #                         "form_key": "i"
-    #                     },
-    #                     "j": {
-    #                         "class": "ListOffsetArray",
-    #                         "offsets": "i64",
-    #                         "content": "int64",
-    #                         "form_key": "j"
-    #                     }
-    #                 }
-    #             }
-    #         },
-    #         "v": {
-    #             "class": "NumpyArray",
-    #             "primitive": "int64",
-    #             "form_key": "v"
-    #         },
-    #         "w": {
-    #             "class": "NumpyArray",
-    #             "primitive": "int64",
-    #             "form_key": "w"
-    #         },
-    #         "x": {
-    #             "class": "NumpyArray",
-    #             "primitive": "int64",
-    #             "form_key": "x"
-    #         }
-    #     }
-    # }
-    # """
-    # )
-
-    builder.begin_list()  # u
-
-    builder.int64(1)  # i
-    builder.begin_list()  # j
-    builder.int64(9)
-    builder.int64(8)
-    builder.int64(7)
-    builder.end_list()  # j
-
-    builder.end_list()  # u
-
-    builder.int64(2)  # v
-    builder.int64(3)  # w
-    builder.int64(4)  # x
-
-    builder.begin_list()  # u
-
-    builder.int64(1)  # i
-    builder.begin_list()  # j
-    builder.int64(9)
-    builder.int64(8)
-    builder.int64(7)
-    builder.end_list()  # j
-
-    builder.end_list()  # u
-
-    builder.int64(2)  # v
-    builder.int64(3)  # w
-    builder.int64(4)  # x
-
-    assert ak._v2.to_list(builder.snapshot()) == [
-        {"u": [{"i": 1, "j": [9, 8, 7]}], "v": 2, "w": 3, "x": 4},
-        {"u": [{"i": 1, "j": [9, 8, 7]}], "v": 2, "w": 3, "x": 4},
-    ]
-
-
```

Comment:
I think, the test file can be deleted - the `ArrayBuilder` part here was to compare it with the `LayoutBuilder`.

Reply:
Okay, I'll do that.